OrbitQt/AccessibilityTests.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QAccessibleWidget>
+#include <QApplication>
+#include <list>
+
+#include "AccessibilityAdapter.h"
+#include "AccessibilityInterfaceMock.h"
+#include "orbitglwidget.h"
+
+namespace orbit_qt {
+
+using orbit_gl::GlAccessibleInterface, orbit_gl::A11yRole, orbit_gl::A11yRect, orbit_gl::A11yState;
+using orbit_gl::TestA11yImpl;
+
+TEST(Accessibility, CreationAndManagement) {
+  TestA11yImpl* obj = new TestA11yImpl(nullptr);
```

Comment:
`unique_ptr` and `.get()`, or `optional` and take the address.

Reply:
Same as for OrbitGlAccessibilityTest.cpp - I explicitly want to test behavior on destruction and I think this makes it clearer that this is happening.

Reply:
Same as there, `.reset()` is really clear enough and I really don't want to see `new` and `delete` unless inevitable.

Reply:
Changed, as commented above